src/app/service/resource/resource.service.ts

```diff
@@ -80,6 +80,7 @@ export class ResourceService extends AbstractService {
           'attr-category': resource['attr-category'] || null,
           'attr-default-order': resource['attr-default-order'] || null,
           'attr-hidden': resource['attr-hidden'] || null,
+          isSpotlight: resource.isSpotlight,
```

Comment:
the name needs to be prefixed with `attr-` in order for the API custom attributes framework to actually work.

name it `attr-spotlight`

Reply:
@frett does it need to be in single quotations?

Reply:
it will be for typescript to not complain